.github/workflows/actions.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [next, qa]
+    tags: ['v*']
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+    - uses: actions/checkout@v4
+    - uses: mamba-org/setup-micromamba@v1
+      with:
+        environment-file: environment.yml
+        cache-environment-key: ${{ runner.os }}-env-${{ hashFiles('**/environment.yml') }}
+        cache-downloads-key: ${{ runner.os }}-downloads-${{ hashFiles('**/environment.yml') }}
+        condarc: |
+          channels:
+            - conda-forge
+            - mcvine
+            - mantid
+    - name: pre-commit
+      run: pre-commit run --all-files
+    - name: Install histogram
+      run: python -m pip install -e . --no-deps
+    - name: Run tests
+      run: xvfb-run --server-args="-screen 0 1920x1080x24" -a python -m pytest --cov --cov-report=xml --cov-report=term --cov-config=.coveragerc
```

Comment:
--cov-config=.coveragerc calls the .coveragerc file which ignores _version.py

Reply:
slug: neutrons/histogram-dev
check here: https://github.com/neutrons/garnet_reduction/blob/next/.github/workflows/garnetreduction.yml#L39C19-L39C50